conf/test.config

```diff
@@ -0,0 +1,8 @@
+params {
+    genomes     = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/genome/genome.fasta'
+    // TODO: Replace with a larger genome as LTRHARVEST does not detect any LTRs on this one
```

Comment:
We need to switch it to a larger genome. I added a large segment of chr1 from Actinida to the nf-core datasets repo here: 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/eukaryotes/actinidia_chinensis/genome/chr1/genome.fasta.gz'

We can switch to it in the future. But we will also need to include the `GUNZIP` module because it is zipped.